addon/components/amount-input/template.hbs

```diff
@@ -1,10 +1,10 @@
-<span class="amount-input__currency {{ if value "black" }}">{{currency}}</span>
+<span class="amount-input__currency {{ if this.value "black" }}">{{this.currency}}</span>
 
 <input
-  class="amount-input__value {{ inputClass }} {{ if disabled "disabled" }}"
+  class="amount-input__value {{ this.inputClass }} {{ if this.disabled "disabled" }}"
```

Comment:
Can't see `disabled` property on the component, is this a named argument `@disabled`?

Reply:
Can we set a `@disabled` in a non octane component ?

Reply:
Ah sorry didn't notice it was a classic component :grimacing: